inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -98,12 +98,77 @@ def article_exists(obj, eng):
     """
     matches = dedupe_list(match(obj.data))
     record_ids = [el['_source']['control_number'] for el in matches]
-    if record_ids:
-        obj.extra_data['record_matches'] = record_ids
-        return True
+    obj.extra_data.setdefault('matches', {})['exact'] = record_ids
```

Comment:
has to be the first record of the list in ``matched.approved``

Reply:
Great!